week-1/K-functions-parameters/exercise5.js

```diff
@@ -1,4 +1,7 @@
 // Declare your function here
+function createLongGreeting(name, age) {
+  return "Hello, my name is" + " " + name + " " + age;
```

Comment:
This will generate a string like this: `Hello, my name is Daniel 30`. I think the greeting can be improved :D

Reply:
@martynamaron , yes it can